news/line79.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* reduce the line width limit to 79
```

Comment:
This would be considered a fix, not a change.  A change is reserved for a change in behavior that a user might need to know about.